pkg/provider/webhook/webhook.go

```diff
@@ -36,7 +36,7 @@ import (
 	esmeta "github.com/external-secrets/external-secrets/apis/meta/v1"
 	"github.com/external-secrets/external-secrets/pkg/provider"
 	"github.com/external-secrets/external-secrets/pkg/provider/schema"
-	"github.com/external-secrets/external-secrets/pkg/template"
+	"github.com/external-secrets/external-secrets/pkg/template/v1"
```

Comment:
I'm not sure whether we want to support the legacy functions. They aren't really relevant because `sprig` funcs are already added.

`pem` or `pkcs12` related support functions aren't really need here.

Reply:
maybe deprecate it and remove when v1alpha1 gets removed?

Reply:
We don't have versioned `SecretStore` providers. I guess what we could do is to *try* executing templates with functions from both template versions - but this is ugly af :smile: 

Webhook provider was released in 0.4.0 and probably doesn't have widespread usage yet. I'd argue that the v1 `template` functions are broken and can be classified as a bug in this provider. I think we should replace them with `template/v2` functions.

Reply:
Oh, yes, that would be ugly af... I haven't quite understood the impact of leaving it there deprecated.

I agree with you. We should upgrade it since the webhook is fairly new. Otherwise it would be the same pain in the ass for users, but for more users.